packages/example/src/components/Transactions/TransactionForm.tsx

```diff
@@ -38,20 +38,6 @@ const StatusBlock = ({ color, text, icon }: StatusBlockProps) => (
   </InformationRow>
 )
 
-const transactionErrored = (transaction: TransactionStatus) => {
-  return 'errorMessage' in transaction
-}
-
-const getTransactionErrorMessage = (transaction: TransactionStatus) => {
-  if ('errorMessage' in transaction) return transaction.errorMessage
-  else return ''
-}
-
-const getTransactionChainId = (transaction: TransactionStatus) => {
```

Comment:
This seems to be useful or might be useful, not sure how others view it, but same value can be gotten directly without a shorthand function...

Maybe the shorthand can be left to users?